run.py

```diff
@@ -99,13 +99,17 @@ def get_iss_cmd(base_cmd, elf, log):
   return cmd
 
 
-def gen(test_list, csr_file, isa, simulator, simulator_yaml, output_dir, sim_only,
-        compile_only, lsf_cmd, seed, cwd, cmp_opts, sim_opts, timeout_s):
+def gen(test_list, csr_file, pass_val, fail_val, csr_end_addr, isa, simulator,
+        simulator_yaml, output_dir, sim_only, compile_only, lsf_cmd, seed,
+        cwd, cmp_opts, sim_opts, timeout_s):
   """Run the instruction generator
 
   Args:
     test_list      : List of assembly programs to be compiled
     csr_file       : YAML file containing description of all CSRs
+    pass_val       : Value that priv. CSR test writes on success
+    fail_val       : Value that priv. CSR test writes on failure
+    csr_end_addr   : Address that priv. CSR test writes to at end of test
```

Comment:
maybe change it to signature_addr? I think this could be used by other tests later.

Reply:
Will do.